salt/config/__init__.py

```diff
@@ -3319,6 +3330,57 @@ def _cache_id(minion_id, cache_file):
         log.error('Could not cache minion ID: {0}'.format(exc))
 
 
+def eval_id_function(opts):
```

Comment:
I went with `call_id_function`. `call_external_id_gen_function` is very descriptive, but was a bit long-winded for my tastes.